docs/modules/tiered-storage/pages/troubleshooting.adoc

```diff
@@ -0,0 +1,22 @@
+= Troubleshooting
+
+== Too Many Open Files
+
+Since Tiered Storage is backed by files on the disk, it increases the open file descriptor count of the operating system by the number of files it uses.
+When Tiered Storage reports "too many open files", the open file descriptors in the operating system reached the maximum open file descriptors setting.
+In this case, Tiered Storage can't open more files, and it may fail to read or write data from, or to the disk.
+Failed writes retry until they succeed to guarantee consistency of the internal state of Tiered Storage, but reads may get reported to the client.
+
+By default, every map with Tiered Storage enabled, and every configured index of these maps use 128 files at maximum.
+
+== Tiered Storage Rejects Storing Big Entries
+
+Storing big entries may get rejected by Tiered Storage in the form of throwing exception with a message like below.
+
+```
+The requested tiered storage record (combined key + value) size: <entry_size> is bigger than the maximum size: 524284. Increase hybrid log page size to store bigger values.
+```
+
+The HybridLog data structure is internally organized in pages and the size of the page defines the maximum record size it can store.
+If this problem is experienced with big entries, increase the page size by setting the `hazelcast.tiered.store.hybridlog.page.size.in.mb`.
+Check the configuration page for description of this property.
```

Comment:
For further information about this property, see xref:tiered-storage:configuration.adoc[].

Reply:
Addressed with https://github.com/hazelcast/hz-docs/pull/1047/commits/b1b0860cd3a96d882f6ce94a84f1779f61070ad3